docs/competition/volunteers/refreshments.md

```diff
@@ -0,0 +1,35 @@
+# Refreshments
+
+It's important that volunteers stay hydrated, which can be difficult for those in roles which don't allow them to move. (Don't give refreshments to competitors).
+
+**In SR2019, the refreshments are in the Volunteer Rest room, which is past the toilets and to the left. Also water points are on both level 2 and 3 in the corner furthest from the toilets.**
+
+It is the responsibility of the Refreshments role to offer and provide refreshments to all volunteers, especially those who can't leave their roles.
+
+To do this, you should grab a tray, fill it with a mix of drinks and food, and go to all volunteer positions, offering refreshments.
+Please also fill some cups with water from a nearby water fountain, and brew coffee/tea if available.
+
+People at the following locations will be unable to leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
+- tech desks
+- reception
+- helpdesk
+- arena: commentator, marshals, head shepherd, score entry
+
+The tech desks roles and commentator are particularly important as they do not rotate between people at all.
+
+## Running low on refreshments
+
+If we're running low on refreshments, it's your responsiblity to either go or find someone to go get more from the nearby shops. There should be an amount of cash by the refreshments to spend to replenish. It's intended to last for the whole competition, so do not spend it all. If the money has ran out, please speak to the Volunteer Manager.
```

Comment:
given how far coop is away from the venue we should try and avoid this trip being made as it is quite a long way away and would take the majority of the slot. A possible solution would be to have them check stock at the beginning of a shift and report if it is running low we can then send a free volunteer to purchase it.

Reply:
Fixed in 547ece8f01e7bbf3865f009d31a49035c29c6c4d